charts/tidb-drainer/values.yaml

```diff
@@ -48,6 +48,22 @@ tlsCluster:
   certAllowedCN: []
   #  - TiDB
 
+# The tls config between drainer and the downstream database server (MySQL/TiDB)
+tlsSyncer:
+  tlsClientSecretName:
```

Comment:
Please add comments to explain what's this field is for and what keys should be included in the secret or add an example command as in L41-L43.

Reply:
Suggest commenting out the new configurations.

Reply:
```suggestion
  # tlsClientSecretName: ""
```